### ISSUE TITLE
Item list

### ISSUE THREAD

**Issue by amiltonwong** [RyanZotti/Self-Driving-Car#82]
Hi, Ryan, 

Could you reply the item list for the self driving toy car , it'll be more helpful if the direct purchase links for the items are included.

**Comment by sprajosh** [comment]
- Toy Car - $30
- Raspberyy Pi - $32
- Raspberyy Pi Protective Case - $10
- Logitech C210 Webcam - $20
- 64GB Class 10 MicroSD card - $15
- External Battery - $10
- Copious amount of Wire - $20
- Soldering Equipment - $25
- UltraSound - $5

I hope this helps

**Comment by ManasKumarPanigrahi** [comment]
Hi Ryan, how can I get to know about the circuit connection?

**Comment by RyanZotti** [comment]
@amiltonwong I purchased about half of the items online, but the other half I bought at a local electronics store (Microcenter, in my case). Some of the items I bought online were extremely cheap but had to be shipped from overseas, and shipping time was 3-4 weeks in some cases. What I learned is that I'm far too impatient to wait multiple weeks to begin tinkering, so I recommend buying everything from your local electronics store and then you can get started the same day. Most of the items are Raspberry Pi peripherals, so find a place that sells the Pi and you can probably find the rest. 

So short answer is I haven't posted links because the online retailers I used simply took too long and I discourage going that route.

**Comment by GoldenWings** [comment]
Hi,
Would you mind telling me the name of ultrasound items to go and search for, because i see ultrasounds probes that are 1000$+ so i am pretty sure i am looking for something else :)
And what kind of Soldering Equipment?, pictures or names would help alot.
Thanks in advance.

**Comment by 0ca** [comment]
I saw some people using these ultrasonic sensors:
https://www.sparkfun.com/products/13959 ($3.95)

**Comment by RyanZotti** [comment]
@0ca Yes, HC-SR04 is what I used. You can buy them from lots of places and they're usually pretty cheap (less than or about $5). 

@GoldenWings I only use one, but that's just because the car I use is very small and can't fit more (the wiring takes up a lot of space). On bigger cars I've seen people use as many as 5. If you put some in the back you can teach the car how to parallel park. I saw this somewhere on YouTube, but I don't have the link available.
